monix-reactive/shared/src/main/scala/monix/reactive/Observable.scala

```diff
@@ -4664,6 +4664,11 @@ abstract class Observable[+A] extends Serializable { self =>
     Task.create { (s, onFinish) =>
       unsafeSubscribeFn(new ForeachSubscriber[A](cb, onFinish, s))
     }
+
+  /** Transforms the source using the given transformer function. */
+  final def transform[B](transformer: Transformer[A, B]): Observable[B] =
```

Comment:
This should probably not be final. This is because it's sometimes cool to override that return type in things that inherit from `Observable`. For example `transform` on a `Subject` could return a `Subject`.